tests/test_twitterCleanup.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, mock
+import datetime
+
+import twitter_cleanup
```

Comment:
I think that, overall, doing `from twitter_cleanup import TwitterCleanup` would grant us better readability in this file.